src/app/settings/settings.component.html

```diff
@@ -0,0 +1,169 @@
+<ion-header class="ion-no-border ion-no-shadow">
+	<ion-toolbar>
+		<ion-buttons slot="start">
+			<ion-back-button></ion-back-button>
+		</ion-buttons>
+		<ion-buttons slot="end">
+			<ion-button data-testid="c-networks__button--menu">
+				<ion-icon
+					class="text-base"
+					color="dark"
+					src="/assets/svg/menu.svg"
+				></ion-icon>
+			</ion-button>
+		</ion-buttons>
+	</ion-toolbar>
+</ion-header>
+
+<ion-content
+	*ngIf="settings$ | async as currentSettings"
+	data-testid="c-settings-page"
+>
+	<ion-header collapse="condense" class="ion-no-border">
+		<ion-toolbar>
+			<ion-title size="large" class="font-black">
+				{{ "SETTINGS_PAGE.SETTINGS" | translate }}
+			</ion-title>
+		</ion-toolbar>
+	</ion-header>
+	<ion-list>
+		<ion-item-group>
+			<ion-item>
+				<ion-label class="font-bold">{{
+					"SETTINGS_PAGE.LANGUAGE" | translate
+				}}</ion-label>
+				<ion-select
+					data-testid="settings__select-language"
+					[value]="currentSettings.language"
+					(ionChange)="updateLanguage($event)"
+					cancelText="{{ 'CANCEL' | translate }}"
+				>
+					<ion-select-option
+						*ngFor="let item of languages | keyvalue"
+						[value]="item.key"
+					>
+						{{ languages[item.key] }}
+					</ion-select-option>
+				</ion-select>
+			</ion-item>
+
+			<ion-item>
+				<ion-label class="font-bold">{{
+					"SETTINGS_PAGE.CURRENCY" | translate
+				}}</ion-label>
+				<ion-select
+					data-testid="settings__select-currency"
+					[value]="currentSettings.currency"
+					(ionChange)="updateCurrency($event)"
+					cancelText="{{ 'CANCEL' | translate }}"
+				>
+					<ion-select-option
+						*ngFor="let item of currencies | keyvalue"
+						[value]="item.key"
+					>
+						{{ item.value | uppercase }}
```

Comment:
Better to also use the key for the currency.

```suggestion
						{{ item.key | uppercase }}
```